core/domain/stats_services.py

```diff
@@ -296,41 +296,35 @@ def update_exp_issues_for_new_exp_version(
         revert_to_version: int|None. If the change is a revert, the version.
             Otherwise, None.
     """
-    # TODO(pranavsid98): Convert below two lines to use get_by_id() instead.
-    exp_issues_model = stats_models.ExplorationIssuesModel.get_model(
-        exploration.id, exploration.version - 1)
-    if exp_issues_model is None:
+    exp_issues = get_exp_issues(exploration.id, exploration.version - 1)
+    if exp_issues is None:
         create_exp_issues_for_new_exploration(
             exploration.id, exploration.version - 1)
         return
-    exp_issues = get_exp_issues_from_model(exp_issues_model)
 
     # Handling reverts.
     if revert_to_version:
-        # TODO(pranavsid98): Convert below two lines to use get_by_id() instead.
-        old_exp_issues_instance = stats_models.ExplorationIssuesModel.get_model(
-            exploration.id, revert_to_version)
-        old_exp_issues = get_exp_issues_from_model(old_exp_issues_instance)
+        old_exp_issues = get_exp_issues(exploration.id, revert_to_version)
         exp_issues.unresolved_issues = old_exp_issues.unresolved_issues
         exp_issues.exp_version += 1
         create_exp_issues_model(exp_issues)
         return
 
     playthrough_ids_by_state_name = collections.defaultdict(list)
 
-    for idx, exp_issue in enumerate(exp_issues.unresolved_issues):
+    for i_idx, exp_issue in enumerate(exp_issues.unresolved_issues):
         keyname = stats_models.ISSUE_TYPE_KEYNAME_MAPPING[exp_issue.issue_type]
         if keyname == 'state_names':
             state_names = exp_issue.issue_customization_args[keyname]['value']
             for state_name in state_names:
                 # Handle exp issues changes for deleted states.
-                exp_issues.unresolved_issues[idx] = (
+                exp_issues.unresolved_issues[i_idx] = (
                     _handle_exp_issues_after_state_deletion(
                         state_name, exp_issue,
                         exp_versions_diff.deleted_state_names))
 
                 # Handle exp issues changes for renamed states.
-                exp_issues.unresolved_issues[idx], playthrough_ids_by_state_name = ( # pylint: disable=line-too-long
+                exp_issues.unresolved_issues[i_idx], playthrough_ids_by_state_name = ( # pylint: disable=line-too-long
```

Comment:
nit: do same thing you've done on lines 342-343 (please only disable pylint if absolutely necessary, this isn't an example where it is)

Reply:
Done.